code/modules/shuttle/special.dm

```diff
@@ -310,6 +312,14 @@
 	name = "fight pit bear"
 	desc = "This bear's trained through ancient Russian secrets to fear the walls of its glass prison."
 	environment_smash = ENVIRONMENT_SMASH_NONE
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/bear/fightpit/Initialize()
+	. = ..()
+	var/multiplier = round(length(SSticker.mode.current_players[CURRENT_LIVING_PLAYERS]) / BASE_BEAR_DIVISOR, 0.1)
+	maxHealth *= multiplier
+	health *= multiplier
+	melee_damage *= multiplier
```

Comment:
Can you make sure its always greater than 0?

Reply:
Also if theres like 5 people on the bears will be 1 hit kills basically

Reply:
Addressed in my review.